src/config.rs

```diff
@@ -96,13 +96,16 @@ pub struct PeerConfig {
     /// If a certificate is specified here, only the specified certificate will be accepted. The system
     /// truststore will not be used.
     pub certificate: Option<String>,
+
+    pub matchkey_encryption_key: Option<String>,
```

Comment:
Can you document this parameter, similar to the comment above (though maybe a lot shorter).